discovery-provider/src/tasks/tracks.py

```diff
@@ -53,6 +54,62 @@ def track_state_update(
     track_contract = update_task.web3.eth.contract(
         address=contract_addresses["track_factory"], abi=track_abi
     )
+
+    cids = []
+    for tx_receipt in track_factory_txs:
+        txhash = update_task.web3.toHex(tx_receipt.transactionHash)
+        for event_type in track_event_types_arr:
+            track_events_tx = getattr(
+                track_contract.events, event_type
+            )().processReceipt(tx_receipt)
+            for entry in track_events_tx:
+                if event_type == track_event_types_lookup["delete_track"]:
+                    continue
+
+                event_args = entry["args"]
+                track_id = (
+                    event_args._trackId if "_trackId" in event_args else event_args._id
+                )
+                blockhash = update_task.web3.toHex(entry.blockHash)
+
+                track_metadata_digest = event_args._multihashDigest.hex()
+                track_metadata_hash_fn = event_args._multihashHashFn
+                buf = multihash.encode(
+                    bytes.fromhex(track_metadata_digest), track_metadata_hash_fn
+                )
+                track_metadata_multihash = multihash.to_b58_string(buf)
+
+                owner_id = event_args._trackOwnerId
+
+                creator_node_endpoint = (
+                    session.query(User.handle, User.creator_node_endpoint)
+                    .filter(User.user_id == owner_id, User.is_current == True)
+                    .first()
+                )[1]
+
+                cids.append([track_metadata_multihash, creator_node_endpoint])
```

Comment:
we should run is_blacklisted_ipld here and we can probably delete it from the code below